pystac_client/item_search.py

```diff
@@ -76,7 +76,7 @@
 
 OPS = list(OP_MAP.keys())
 
-DEFAULT_LIMIT_AND_MAX_ITEMS = 100
+DEFAULT_LIMIT = 100
```

Comment:
Technically should alias this, since `DEFAULT_LIMIT_AND_MAX_ITEMS` is in the public API.